packages/styles/skip-link.css

```diff
@@ -1,19 +1,17 @@
 :root {
-  --skip-link-background-color: rgba(11, 14, 17, 0.9);
+  --skip-link-background-color: rgba(255, 255, 255, 0.95);
```

Comment:
Did you check with @awpearlm on this? The UX isn't clear, the notes say it's 90% opacity but the actual spec shows 95%.

Reply:
I don't know if anything needs to change where, just want to validate that 95% is correct.

Reply:
I was just going off the uxpin notes.